controllers/api/information.js

```diff
@@ -24,19 +24,11 @@ router.get("/", async (req, res) => {
   }
 });
 
-router.post("/", async (req, res) => {
+router.put("/", async (req, res) => {
```

Comment:
Why are we removing POST route? Should we not be able to create information initially and then edit it later?